src/TestSuite/ConnectionHelper.php

```diff
@@ -74,6 +74,7 @@ public static function enableQueryLogging(?array $connections = null): void
                 $connection->getDriver()->log($message) === false
             ) {
                 $connection->getDriver()->setLogger(new QueryLogger());
+                $connection->getDriver()->enableQueryLogging();
```

Comment:
Won't this change/break logging configurations that are currently working? We could have `setLogger()` also set `logQueries` to true.

Reply:
seems reasonable to always enable query logging when a logger instance is set